### PR TITLE
completing the challenge

### PR DESCRIPTION
**Pull request by m33k3r**
Completing the first challenge and requesting a code review

### CODE REVIEW

File: challenge.rb [hunk 1]
```diff
@@ -1,26 +1,26 @@
 
 def capitalize_each_string(input)
-  #implement your solution here
+  input.map(&:capitalize)
```
Comment (SonofNun15) [inline challenge.rb]: Nice. I like the method reference here.

**Comment by m33k3r** [comment]
Josh Graber Approved

File: challenge.rb [hunk 2]
```diff
@@ -1,26 +1,26 @@
...
 end
 
 def fetch_the_dog(input)
-  #implement your solution here
+  input.grep 'dog'
 end
 
 def no_dogs_allowed(input)
-  #implement your solution here
+  input.find_all {|i| i.downcase != 'dog'}
```
Comment (sjreich) [inline challenge.rb]: `reject` is a nice one here

File: challenge.rb [hunk 3]
```diff
@@ -1,26 +1,26 @@
...
 end
 
 def fetch_the_first_two(input)
-  #implement your solution here
+  input.take(2)
 end
 
 def fetch_CD_animals(input)
-  #implement your solution here
+  input.select {|i| i.start_with? 'c', 'd'}
```
Comment (sjreich) [inline challenge.rb]: nice find.

**Pull request closed without merging**